packages/run-protocol/src/vaultFactory/liquidateIncrementally.js

```diff
@@ -269,11 +280,13 @@ const start = async zcf => {
     trace('exit seat');
   };
 
+  /**
+   * @type {ERef<Liquidator>}
+   */
```

Comment:
one line?

Reply:
Deferring to https://github.com/Agoric/agoric-sdk/pull/4906